.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Fixes: #
+
+## Summary
+- Summary of the solution to fix the issue.
+- Include details of design or implementation choices made, tradeoffs considered, etc.
+- Link to design document (when applicable)
+
+## Type of Changes
+- [ ] New Feature
+- [ ] Bug-fix
+- [ ] Enhancement
+
+## Checklist
+- [ ] Implementation
+- [ ] Documentation
+- [ ] Tests
+
+## Test Results
+- Results of tests done (manual or automated) to verify the changes made.
```

Comment:
Since we are a system where performance and memory foot print is critical, i would like to see a section on "benchmark results". How easy is it for us to run benchmark on new features atm? @JyotinderSingh